configs/config.yaml

```
@@ -49,4 +49,43 @@ firecrawl:
 logging:
   level: "debug"  # Changed from "warn" to "debug" for troubleshooting captcha issues
   format: "json"
-  output: "stdout" 
\ No newline at end of file
+  output: "stdout"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)